docs: Overhaul Explore Profiles documentation (#63)
grafana/explore-profiles: docs/sources/_index.md

title: Explore Profiles
menuTitle: Explore Profiles
weight: 100
hero:
  title: Explore Profiles
  level: 1
  width: 100
  height: 100
  description: Use Explore Profiles to investigate and identify issues using profiling data.
cards:
  title_class: pt-0 lh-1
  items:
    - title: Concepts
      href: ./concepts/
      description: Learn the concepts you need to understand to use profiling.
      height: 24
    - title: Get started
      href: ./get-started/
      description: How do you use profiling data to investigate an issue? Start here.
      height: 24
    - title: Determine your use case
      href: ./troubleshooting/
      description: Find solutions to common issues you might encounter when using Explore Logs.
      height: 24
    - title: Choose a view
      href: ./choose-a-view/
      description: Which exploration type view best fits the issue you are investigating?
      height: 24
    - title: Investigate trends and spikes
      href: ./investigate/
      description: Use your profiling data to identify issues and determine the root cause.
      height: 24
    - title: Changelog
      href: https://github.com/grafana/explore-profiles/blob/main/CHANGELOG.md

>       description:  Learn about the updates, new features, and bugfixes in this version.

Author: ```suggestion
      description:  Learn about the updates, new features, and bugfixes in this version.
      height: 24
```

  FULL_PRODUCT_NAME: Grafana Explore Profiles
  PRODUCT_NAME: Explore Profiles
description: Learn how to use Explore Profiles to understand and troubleshoot
  your applications and services.
keywords:
  - Explore Profiles
  - Profiles
title: Explore Profiles
menuTitle: Explore Profiles
weight: 100
hero:
  title: Explore Profiles
  level: 1
  width: 100
  height: 100
  description: Use Explore Profiles to investigate and identify issues using profiling data.
cards:
  title_class: pt-0 lh-1
  items:
    - title: Concepts
      href: ./concepts/

>       description: Learn the concepts you need to understand to use profiling.

Reviewer: ```suggestion
      description: Learn the concepts you need to use profiling.
```

Author: Good edit.

      height: 24
    - title: Get started
      href: ./get-started/
      description: How do you use profiling data to investigate an issue? Start here.
      height: 24
    - title: Determine your use case
      href: ./troubleshooting/

>       description: Chones one of the wwo most common use cases to guide your exploration.

Reviewer: "Choose"? Also, do you mean "one of the two"?

Author: Yes :)  ERF. I should have run spell check... 
```suggestion
      description: Choose one of the two most common use cases to guide your exploration.
```
